fdbserver/QuietDatabase.actor.cpp

```diff
@@ -735,7 +740,7 @@ ACTOR Future<Void> waitForQuietDatabase(Database cx,
                                         int64_t maxDataDistributionQueueSize = 0,
                                         int64_t maxPoppedVersionLag = 30e6,
                                         int64_t maxVersionOffset = 1e6) {
-	state QuietDatabaseChecker checker;
+	state QuietDatabaseChecker checker(isBuggifyEnabled(BuggifyType::General) ? 1500.0 : 1000.0);
```

Comment:
I don't see any precedent for this (i.e., testing isBuggifyEnabled()) - is this okay?. The intent is to avoid lowering the bar for tests if buggify is not 'on' unless we have reason to do that. Looking for feedback here..

Reply:
LGTM